Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -39,6 +40,8 @@ namespace MultiplayerSample
 
     void NetworkMatchComponent::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
     {
+        RoundRestTimeRemainingAddEvent(m_restTimeRemainingChangedHandler);
```

Comment:
Does this need to be limited to the server only?

Reply:
Client only. Fixed!